.github/workflows/admin_tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Admin tests
+
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  tests:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-latest]
+        node-version: [12.x, 14.x]
```

Comment:
What's the idea behind doing 2 versions here?

Reply:
1. Node version used for development
2. the latest version of Node

Reply:
but how is 2) relevant here? Since we're not using it

Reply:
using only 12.x now